### PR TITLE
Reference Arcade templates locally

### PR DESCRIPTION
**Pull request by chcosta**
Copy eng\common folder from https://github.com/dotnet/arcade/tree/master/eng/common so that any potential volatile changes in Arcade don't instantly break this repo.  In the near future, dependency flow will be the method by which eng\common will stay in sync with master (dependency flow will cause PR's to be created if Arcade publishes changes to these files).  In the mean time, this repo will need to manually take updates to eng\common if desired.

@livarcocc

### CODE REVIEW

**Comment by chcosta** [comment]
@livarcocc , have the webhooks for this repo been updated after the dotnet -> dnceng transition?  I don't see the VSTS public CI gate firing in this PR

**Comment by livarcocc** [comment]
Done.

File: eng/common/native/install-cmake.ps1 [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<#
```
Comment (livarcocc) [inline eng/common/native/install-cmake.ps1]: Do we need this if it is not used in the repo?

File: eng/common/build.ps1 [hunk 2]
```diff
@@ -0,0 +1,320 @@
+[CmdletBinding(PositionalBinding=$false)]
```
Comment (livarcocc) [inline eng/common/build.ps1]: Shouldn't we delete the old files for these that are currently under build/?

**Pull request merged**